salt/metalk8s/_internal/init.sls

```diff
@@ -0,0 +1,14 @@
+#
+# Requirements states
+#
+# 1 directory per requirement.
+# Each requirement should contain (at least):
+# ===========================================
+#
+# * installed   -> install the requirement
```

Comment:
That's not true, preflight directory does not contain any installed or removed.

Reply:
Yes agree but just not wanted to do all the preflight in this PR but btw we have this ticket #987